acsg.js

```diff
@@ -218,35 +218,35 @@ acsg.Browser = (function () {
 acsg.CLI = (function () {
 
   var CLI = function (opts) {
-      if (!(this instanceof CLI)) {
-          return new CLI(opts)
-      }
+    if (!(this instanceof CLI)) {
+      return new CLI(opts)
+    }
 
-      this.opts = opts
-      this._performance = require('perf_hooks')
+    this.opts = opts
+    this._performance = require('perf_hooks')
   }
 
   CLI.prototype.now = function () {
     return this._performance.performance.now()
   }
 
-  CLI.prototype.draw = function (position, color) {
+  CLI.prototype.draw = function () {
     // Noop
```

Comment:
I wasn't quite sure what to do here. On the one hand, the unused args trigger the linter warnings. On the other hand, the method signatures _are_ part of the interface of the "duck type" implemented by `Browser` and `CLI`, and in both cases they'll be called with the arguments even if CLI doesn't do anything with them. My instinct was to favor matching the interface over the linting concerns.

Reply:
I was in the same boat, but because JS doesn't care if you pass extra arguments to a function, I erred on the side of making the linter happy. If we do want to leave the arguments in, then we can just add a comment telling the linter to ignore the unused argument on the next line:

    CLI.prototype.draw = function (position, color) { // eslint-disable-line no-unused-vars

Reply:
OK, I've come around to just embracing the JS nonchalance here ("I'm clearly not doing anything, so pass me anything or nothing... I don't care.")